### ISSUE TITLE
apollo-server-express 2.11 + graphql 15.0 = error: Must provide Source. Received: undefined.

### ISSUE THREAD

**Issue by NGPixel** [apollographql/apollo-server#3952]
Upon upgrading the graphql package to `15.0.0`, the package `apollo-server-express` throws an error as soon as it's being included. Reverting back to `14.6.0` fixes the issue.

```
 error: Must provide Source. Received: undefined.
```

**Steps to reproduce:**
- Install latest `apollo-server-express` package (`2.11.0`)
- Install latest `graphql` package (`15.0.0`)
- Run sample code (index.js):
```
const apollo = require('apollo-server-express')
```
- Process will crash immediately with above error.

**Comment by Vultraz** [comment]
Also encountered this. It seems `apollo-server-express` (and by extension, `apollo-server-core` rely on a slightly outdated version of `graphql-tools`: https://github.com/graphql/graphql-js/issues/2503 (4.0.0). Forcing resolution for `graphql-tools": ^4.0.6` should work, I think.

**Comment by Glen-Moonpig** [comment]
Same issue with apollo-server-lambda 2.13.0 + graphql 15.0.0

**Comment by clok** [comment]
Also seen with `apollo-server-hapi` and graphql v15. Has to drop down to graphql v14 latest.

**Comment by Vultraz** [comment]
Really hope they fix this for server v3 at least.

**Comment by prabhuignoto** [comment]
having the same issue, down graded to graphql@14

**Comment by lolgesten** [comment]
Still the same in apollo-server 2.16.1.

We get multiple graphql versions cause `@types/graphql-upload@8.0.3` drags in graphql@14.

The double graphql version problem were also found here https://github.com/graphql/graphql-js/issues/2503

```
$ npm ls graphql
lookback-ultron@0.0.1 /Users/martin/dev/lk/ultron
├─┬ apollo-server@2.16.1
│ └─┬ apollo-server-core@2.16.1
│   └─┬ @types/graphql-upload@8.0.3
│     └── graphql@14.7.0 
└── graphql@15.3.0 
```

**Comment by sannamar** [comment]
This is still an issue with apollo-server 2.19.2.

Running `npm ls graphql` outputs the following: 
```
├─┬ apollo-server@2.19.2
│ └─┬ apollo-server-core@2.19.2
│   └─┬ @types/graphql-upload@8.0.3
│     └── graphql@14.7.0
└── UNMET PEER DEPENDENCY graphql@15.5.0
```

**Comment by glasser** [comment]
I'm going to close this as duplicate of #4865 but I'll make sure to note that it might be slightly different than just a peer dep issue.

**Comment by glasser** [comment]
Apollo Server v2.21.0 will fix this.  I published alpha packages with version v2.21.0-alpha.0.  I'd love some folks to try the alpha and let me know if it works for you! Both folks who actually use the upload feature and those who don't! It's currently in the release-2.21.0 branch, being tracked at https://github.com/apollographql/apollo-server/pull/4922
